source/stylist/fortran.py

```diff
@@ -119,20 +121,28 @@ class MissingImplicit(FortranRule):
     don't.
     """
 
-    def __init__(self, default='none', require_everywhere=False) -> None:
+    def __init__(self,
+                 default: str = 'none',
+                 require_everywhere: Optional[bool] = False) -> None:
         """
-        Constructor taking a default implication.
-
-        Todo: This rule was designed to check merely for the presence of an
-        "implicit" statement but to leave the choice of implication to the
-        user. It might also be desirable to allow this rule to enforce a
-        specific implication. In which case the "default" argument may need a
-        different name. A switch would be necessary to flip between "any" and
-        "specific" modes.
-
-        Todo: At the moment the default is not used. In the future we may want
-        to allow rules to enforce style rather than simply reporting it. In
-        this case the default would be used where none is present.
+        :param default: Impose this "implicit" where none is found.
```

Comment:
I would suggest using code mark-up for `implicit`, similar to how code is highlighted in other docstrings.

Reply:
Done

Reply:
Also on line 120

Reply:
Also done